### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by sagieramos**
### We have added the following features
- Popup Window
  - When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
  - When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: js/popup.js [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+  objItem.tag.dk.forEach((tagItem) => {
+    const li = document.createElement('li');
+    li.textContent = tagItem;
+    ulPopupTagsDk.appendChild(li);
+  });
+
+  const ulPopupBtn = document.createElement('ul');
+  ulPopupBtn.className = 'popup-btn';
+
+  const btnLive = document.createElement('button');
```
Comment (andersonlebon) [inline js/popup.js:255]: - [ ] kindly note this button should be a link that will link to the live version of the project and the link should be found in the popup data object(obj)

File: js/popup.js [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+  btnLive.className = 'live l-btn';
+  btnLive.textContent = 'See live';
+
+  const btnSource = document.createElement('button');
```
Comment (andersonlebon) [inline js/popup.js:258]: - [ ] kindly note this button should be a link that will link to the code source of the project and the link should be found in the popup data object(obj)

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @sagieramos 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your design is njce ✔️ 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @sagieramos, and @richie1988👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**